### ISSUE TITLE
[Bug]: Loss chart is always empty for ACT models (frontend reads train_loss, ACT logs train/loss_step)

### ISSUE THREAD

**Issue by Timshen891203** [open-edge-platform/physical-ai-studio#871]
### Description

The Model Metrics loss chart renders no line for ACT training jobs. The chart scaffolding appears (axes, gridlines, correct X-axis step ticks) but the Y axis has no ticks and no area/line is drawn. The LOSS column in the model list shows a correct value, so the data is reaching the client.

This affects ACT only. Other policies are unaffected.

### Steps to Reproduce

1. Train an ACT model via the UI (any dataset).
2. Go to Models → expand the running/finished job → Model Metrics tab.
3. The chart is empty.

### Expected Behavior

The per-step training loss is plotted against step.

### Actual Behavior

: X axis shows correct step values (e.g. 49 … 3949), Y axis has no ticks, no line is drawn.

<img width="1917" height="802" alt="Image" src="https://github.com/user-attachments/assets/7e8e2bf8-7292-40de-a038-251a690530c4" />

### Component

Library (VLA framework)

### Environment

```markdown
Ubuntu 20.04 
Docker XPU profile 
commit 1e213180
lerobot 0.6.0
```

### Logs / Error Output

```shell
$ head -1 metrics.csv
epoch,step,train/loss_step
0,49,4.1512346267700195
0,99,4.300111293792725
```

### Additional Context

Root cause — application/ui/src/routes/models/metrics.tsx:

ACT logs the per-step value as train/loss_step (library/src/physicalai/policies/act/policy.py:356), and logs train/loss with on_step=False, on_epoch=True — so train/loss is absent from the CSV until the first epoch ends. The streamed object therefore has no train_loss key: undefined !== null is true, so every row passes the filter with y === undefined. X renders from row.step; Y cannot compute a domain.

Why ACT only: other policies (smolvla, pi05, pi0, groot, lerobot) call self.log("train/loss", ...) inside training_step, where Lightning defaults to on_step=True — so train_loss is present per step and their charts render.

Suggested fix: use train_loss_step (the helper is already named filterLossStepMetrics). Consider != null to also guard undefined.

Related but distinct: #508 (loss graph feature request), #845 (readability of values).
